reference-architectures/sandboxes/examples/gcp-sandboxes/README.md

```diff
@@ -0,0 +1,121 @@
+# Overview
+
+This directory contains Terraform configuration files to quickly set up the the
+sandboxes system project. This example is a good entry point for testing the
+reference architecture and learning how it can be incorportated into your own
+infrastructure as code processes.
+
+## Architecture
+
+![architecture-per-project](../../resources/high-level-arch.png)
+
+For an explanation of the components of the sandboxes reference architecture and
+the interaction flow, please read the
+[main Architecture section](../README.md#architecture).
+
+## Before you begin
+
+In this section you prepare a folder for deployment.
+
+1.  Open the [Cloud Console][cloud-console]
+2.  Activate [Cloud Shell][cloud-shell] \
+    At the bottom of the Cloud Console, a [Cloud Shell][cloud-shell-features]
+    session starts and displays a command-line prompt. Cloud Shell is a shell
+    environment with the Cloud SDK already installed, including the `gcloud`
+    command-line tool, and with values already set for your current project. It
+    can take a few seconds for the session to initialize.
+
+3.  In Cloud Shell, clone this repository
+
+    ```sh
+    git clone https://github.com/GoogleCloudPlatform/platform-engineering.git
+    ```
+
+4.  Export variables for the working directories
+
+    ```sh
+    export SANDBOXES_DIR="$(pwd)/reference-architectures/examples/gcp-sandboxes"
+    export SANDBOXES_CLI="$(pwd)/reference-architectures/examples/cli"
+    ```
+
+### Preparing the Sandboxes Folder
+
+In this section you prepare your enviroment for deployment of the system
+project.
+
+1.  Go to the [Manage Resources][manage-resources] page in the Cloud Console in
+    the IAM & Admin menu.
+
+2.  Select Create folder, then choose Folder.
```

Comment:
All references to button captions should be bold: **Create folder**

Prefer "Click" over "Select" unless it's some kind of list or menu